scripts/next_step_assignment_gx.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+from EPPs.common import StepEPP
+from pyclarity_lims.entities import Protocol
+
+
+class AssignNextStep(StepEPP):
+    _use_load_config = False  # prevent the loading of the config file
+    """
+    This script assigns Next Step as either "remove from protocol" if SSQC Results = "PASSED" or "request manager 
+    review" if SSQC Reults contains "FAILED"
+    """
+
+    def _run(self):
+
+        # obtain the actions of the step then creates a StepActions entity for the current step
+        actions = self.process.step.actions
+
+        # obtain the next actions in the step then creates a list of dict for next_actions for the step
+        next_actions = actions.next_actions
```

Comment:
I think that by now these two lines/comments are superfluous.
The for loop should be rewritten
```python
for next_action in actions.next_actions:
    ...
```